dom/dom_c.php

```diff
@@ -179,35 +180,38 @@ public function insertBefore(
     /**
      * Replaces a child
      * @link https://php.net/manual/en/domnode.replacechild.php
+     * @template TNode of DOMNode
      * @param DOMNode $node <p>
      * The new node. It must be a member of the target document, i.e.
      * created by one of the DOMDocument->createXXX() methods or imported in
      * the document by .
      * </p>
-     * @param DOMNode $child <p>
+     * @param TNode $child <p>
      * The old node.
      * </p>
-     * @return DOMNode|false The old node or false if an error occur.
+     * @return TNode|false The old node or false if an error occur.
      */
     public function replaceChild(DOMNode $node, DOMNode $child) {}
 
     /**
      * Removes child from list of children
      * @link https://php.net/manual/en/domnode.removechild.php
-     * @param DOMNode $child <p>
+     * @template TNode of DOMNode
+     * @param TNode $child <p>
      * The removed child.
      * </p>
-     * @return DOMNode If the child could be removed the functions returns the old child.
+     * @return TNode|false If the child could be removed the functions returns the old child.
```

Comment:
This returns the child removed (which is the one in param)
AND `false` was missing

https://www.php.net/manual/en/domnode.removechild.php